Vostok.ClusterConfig.Client/ClusterConfigClient.cs

```diff
@@ -312,5 +320,11 @@ private void PropagateError([NotNull] Exception error, bool alwaysPushToObservab
             if (pushedToSource || alwaysPushToObservable)
                 Task.Run(() => stateObservable.Error(error));
         }
+
+        // ReSharper disable InconsistentNaming
```

Comment:
Why did the constants move to the bottom? Is this the behaviour of auto formatting with our code style?

Reply:
Yes, seems effect of `ReSharper disable InconsistentNaming`.